salt/metalk8s/salt/master/files/salt-master-manifest.yaml.j2

```diff
@@ -118,7 +118,7 @@ spec:
     - name: run
       hostPath:
         path: '/var/run/salt'
-        type: Directory
+        type: DirectoryOrCreate
```

Comment:
Not super fond of that. Can't we/shouldn't we instead create the directory on the host as part of the Salt state which lays out the manifest?

Reply:
Follow-up: `/var/run` is `tmpfs` so not persisted. Hence, `DirectoryOrCreate` *is* the right approach.

Reply:
For this specific one, no, we can't since it's part of tmpfs `/var/run`